Source/WebInspectorUI/UserInterface/Views/SourceCodeTextEditor.js

```diff
@@ -748,9 +748,16 @@ WI.SourceCodeTextEditor = class SourceCodeTextEditor extends WI.TextEditor
 
         let locations = await script.breakpointLocations(startPosition, endPosition);
         for (let location of locations) {
+            let position = this.originalPositionToCurrentPosition(location.position());
+
+            // Don't show an inline widget when there is only one breakpoint location on the line
+            // and it's at the start of the line.
+            if (locations.length === 1 && position.lineNumber === lineNumber && !this.line(lineNumber).slice(0, position.columnNumber).trim().length)
```

Comment:
The last check, while likely to be true, is not a given I think... I think a comment before an expression would fail this check, but still only have a single breakpoint on the line. e.g.
```
/* I'm an oddly-placed comment */ doAllTheThings()
```